.env.example

```diff
@@ -0,0 +1,30 @@
+# needs to be real so create a personal magic.link account or use the staging key
```

Comment:
Could we use `.env.tpl` like we do in web3 side?

https://github.com/web3-storage/web3.storage/blob/main/.env.tpl

Reply:
sure, good call